exchange/bidder.go

```diff
@@ -76,6 +76,10 @@ const (
 	Gzip string = "GZIP"
 )
 
+var (
+	tmaxTimeoutErr = errors.New("exceeded tmax duration")
+)
```

Comment:
Good use of the error pattern. Nitpick on the naming, it's common to start with "Err". Consider the name `ErrTmaxTimeout`.

Reply:
https://github.com/prebid/prebid-server/pull/2787/commits/cb18ddff6ee148d7cba9a783cbb6b486cdd889f8